docs/manage/_get_support.rst

```diff
@@ -84,9 +84,10 @@ Open the ``.kolibri`` folder inside the :ref:`Home <home>` folder of the device
 About IP addresses
 ------------------
 
-		* ``0.0.0.0`` = A special IP address on the **server** (your device running Kolibri and "serving" its content to others in the local network), which actually means "all available IP addresses". It's a kind of alias. But accessing ``0.0.0.0`` from another computer doesn't make sense and doesn't work. By default, Kolibri will serve on ``0.0.0.0``, which essentially means all IP addresses that are available on the device will render Kolibri accessible.
-		* ``127.0.0.1`` = A device's local IP address, meaning "myself". Some people joke and say "There's no place like 127.0.0.1", meaning "there's no place like home" :) This can be used on the serving device itself to test that Kolibri is running, in case you need a failsafe way of checking that Kolibri is in fact running and responsive.
-		* ``192.x.y.z`` = Addresses starting with ``192`` are local network IP addresses. The same thing can be said about ``10.x.y.z``. The address that you wanna use to enter on the clients/tablets in order to contact the server will in most cases start with ``192`` or ``10``.
-		* Port number: Kolibri runs on port ``8080``. When you access something on an IP address, you need a port. Ports can be open or closed on the server, but they can also be regulated by firewall rules on the way. ``http://`` <- this is the protocol that the browser reads out from the "URL", which is just some text that describes Kolibri.
-		* ``http://192.168.1.1:8080`` means: "Connect to IP address ``192.168.1.1`` on port ``8080`` with the HTTP protocol". The browser will the continue to try to reach this address, but may fail for instance if Kolibri isn't running, or if a step along the way blocks access.					
+.. ``0.0.0.0`` = A special IP address on the **server** (your device running Kolibri and "serving" its content to others in the local network), which actually means "all available IP addresses". It's a kind of alias. But accessing ``0.0.0.0`` from another computer doesn't make sense and doesn't work. By default, Kolibri will serve on ``0.0.0.0``, which essentially means all IP addresses that are available on the device will render Kolibri accessible.
+
+* ``127.0.0.1`` IP address, or ``localhost``, is device's own IP address. You can use it in the browser on the device where Kolibri is running to make sure it is working correctly. 
+* Aside from its own `localhost <https://en.wikipedia.org/wiki/Localhost>`_ address, device running Kolibri also has an IP address like ``192.x.y.z`` or ``10.x.y.z``, under which it is recognized by other devices in the same local network. That is the IP address that you need to use in the :ref:`browsers on client devices <access_LAN>` (learner tablets or computers), to connect with Kolibri server.
```

Comment:
Aside from its own localhost address, **[a]** device running Kolibri also has an **[external]** IP address like `192.x.y.z` or `10.x.y.z`

Reply:
Also, I wonder if it's helpful or confusing to show `x.y.x`?

Maybe just use more realistic complete examples like `192.168.1.1` and `10.0.0.3`?

Reply:
Instead of `.x.y.z` (which I think is pretty okay?), it could also be `.*.*.*` - might also be a bit conventional, not sure. But conventions won't help a lot in this case, anyways.

I don't think the concrete numbers are a good idea. This guide is supposed to be of slight help for someone who doesn't know what IP addresses are.. but by showing specific IP addresses we risk that people think they have to assign one of those particular addresses. And then that could lead the user into a series of complicated misunderstandings.

If I understand the IPv4 system correctly, it is safe to assume that any local and externally reachable IP address will be called either 192.something or 10.something. So if we can just _somehow_ communicate to people that "if you see 4 numbers separated by dots and the first one is either 192 or 10, then it's the external IP address" :)

Reply:
we use concrete numbers below, so either way we should be consistent

Reply:
I kept the abstract pattern, but used * as suggested. Concrete numbers in the last item are within quotes, to indicate that it's a real-life example.